.github/workflows/push-updates.yml

```diff
@@ -0,0 +1,29 @@
+# name: Sync parent repo with submodule
```

Comment:
What is this for?

Reply:
As mentioned in this issue by @teodutu it's a template for a GitHub action to push the changes to our website.

[1]: https://github.com/security-summer-school/template-public/issues/11

Reply:
I will close this since it's not an issue.